common/Field_Dimensions.cpp

```diff
@@ -9,4 +9,7 @@ const Field_Dimensions Field_Dimensions::Double_Field_Dimensions(
     1.000f, 0.500f, 1.000f, 0.500f, 10.400f, 7.400f);
 
 Field_Dimensions Field_Dimensions::Current_Dimensions =
-    Field_Dimensions::Double_Field_Dimensions;
\ No newline at end of file
+    Field_Dimensions::Double_Field_Dimensions;
+
+const Field_Dimensions Field_Dimensions::Default_Dimensions =
```

Comment:
It may make more sense then to initialize these with: default = double; current = default;